x-pack/plugins/apm/ftr_e2e/cypress/integration/power_user/feature_flag/infrastructure.spec.ts

```diff
@@ -14,17 +14,17 @@ const start = '2021-10-10T00:00:00.000Z';
 const end = '2021-10-10T00:15:00.000Z';
 
 describe('Infrastracture feature flag', () => {
-  before(async () => {
-    await synthtrace.index(
+  before(() => {
+    synthtrace.index(
```

Comment:
We should no longer use `await` before `synthtrace.clean` or `synthtrace.index` since they are Cypress tasks handled by cypress.